Error inflating class RadioButton when opening Subscriptions filter dialog (building from develop)

# Checklist
<!-- Place an x in the boxes to tick them: [x] -->

- [x] I have used the search function to see if someone else has already submitted the same bug report.
- [x] I will describe the problem with as much detail as possible.
- [x] If the bug only to occurs with a certain podcast, I will include the URL of that podcast.

# System info
<!-- The following information is very important to fill out because some bugs may only occur on certain devices or versions of Android. -->

**App version**: 2.0.0-RC7 (develop branch retrieved 2020-09-17 from github, freeDebug build, commit b80978c40ac22bdaaea528c3908c5cf347088562)
<!-- The latest version may be different depending on your device. You can find the version in AntennaPod's settings. -->

**Android version**: 6.0.1 stock

**Device model**: Nexus 5

# Bug description

When I open the filter dialog in a subscribed podcast, I get an exception.

**Steps to reproduce**:
1. Navigate to Subscriptions.
2. Pick a subscribed podcast.
3. Click the "Filter" button.

I don't know if this is a problem with how I'm building or a real problem. I am being sure to clean before I build, though. Running on AndroidStudio with Run->Run 'app'.

Versions:
- AndroidStudio 4.0.1
- Android Gradle plugin version 4.0.1
- Gradle 6.3

**Expected behaviour**:
<!-- After following the steps, what did you think AntennaPod would do? -->
I expected the filter dialog to open.

**Current behaviour**:
<!-- What did AntennaPod do instead? Screenshots might help. Usually, you can take a screenshot of your smartphone by pressing *Power* + *Volume down* for a few seconds. -->
AntennaPod terminated with "AntennaPod has stopped."

**First occurred**: I just started looking at the 2.0.0 source a couple of days ago.

**Environment**: I installed on top of a 1.8.2 debug build.

**Stacktrace/Logcat**:
<!-- If you are experiencing a crash, including the stacktrace will likely get it fixed sooner. AntennaPod has an `export logs` feature for this. -->
```
09-17 22:29:34.828 23478-23478/de.danoeh.antennapod.debug D/UserPreferences: Using default data folder
09-17 22:29:34.870 23478-23478/de.danoeh.antennapod.debug E/AndroidRuntime: FATAL EXCEPTION: main
    Process: de.danoeh.antennapod.debug, PID: 23478
    android.view.InflateException: Binary XML file line #23: Binary XML file line #23: Error inflating class RadioButton
        at android.view.LayoutInflater.inflate(LayoutInflater.java:539)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:374)
        at de.danoeh.antennapod.dialog.FilterDialog.openDialog(FilterDialog.java:41)
        at de.danoeh.antennapod.menuhandler.FeedMenuHandler.showFilterDialog(FeedMenuHandler.java:116)
        at de.danoeh.antennapod.menuhandler.FeedMenuHandler.onOptionsItemClicked(FeedMenuHandler.java:76)
        at de.danoeh.antennapod.fragment.FeedItemlistFragment.onOptionsItemSelected(FeedItemlistFragment.java:271)
        at androidx.fragment.app.Fragment.performOptionsItemSelected(Fragment.java:2830)
        at androidx.fragment.app.FragmentManager.dispatchOptionsItemSelected(FragmentManager.java:2717)
        at androidx.fragment.app.FragmentController.dispatchOptionsItemSelected(FragmentController.java:412)
        at androidx.fragment.app.FragmentActivity.onMenuItemSelected(FragmentActivity.java:389)
        at androidx.appcompat.app.AppCompatActivity.onMenuItemSelected(AppCompatActivity.java:228)
        at androidx.appcompat.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:109)
        at androidx.appcompat.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:109)
        at androidx.appcompat.app.ToolbarActionBar$2.onMenuItemClick(ToolbarActionBar.java:64)
        at androidx.appcompat.widget.Toolbar$1.onMenuItemClick(Toolbar.java:207)
        at androidx.appcompat.widget.ActionMenuView$MenuBuilderCallback.onMenuItemSelected(ActionMenuView.java:779)
        at androidx.appcompat.view.menu.MenuBuilder.dispatchMenuItemSelected(MenuBuilder.java:834)
        at androidx.appcompat.view.menu.MenuItemImpl.invoke(MenuItemImpl.java:158)
        at androidx.appcompat.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:985)
        at androidx.appcompat.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:975)
        at androidx.appcompat.widget.ActionMenuView.invokeItem(ActionMenuView.java:623)
        at androidx.appcompat.view.menu.ActionMenuItemView.onClick(ActionMenuItemView.java:151)
        at android.view.View.performClick(View.java:5204)
        at android.view.View$PerformClick.run(View.java:21153)
        at android.os.Handler.handleCallback(Handler.java:739)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:148)
        at android.app.ActivityThread.main(ActivityThread.java:5417)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
     Caused by: android.view.InflateException: Binary XML file line #23: Error inflating class RadioButton
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:782)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:704)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:835)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:838)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:838)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:798)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:423) 
        at android.view.LayoutInflater.inflate(LayoutInflater.java:374) 
        at de.danoeh.antennapod.dialog.FilterDialog.openDialog(FilterDialog.java:41) 
        at de.danoeh.antennapod.menuhandler.FeedMenuHandler.showFilterDialog(FeedMenuHandler.java:116) 
        at de.danoeh.antennapod.menuhandler.FeedMenuHandler.onOptionsItemClicked(FeedMenuHandler.java:76) 
        at de.danoeh.antennapod.fragment.FeedItemlistFragment.onOptionsItemSelected(FeedItemlistFragment.java:271) 
        at androidx.fragment.app.Fragment.performOptionsItemSelected(Fragment.java:2830) 
        at androidx.fragment.app.FragmentManager.dispatchOptionsItemSelected(FragmentManager.java:2717) 
        at androidx.fragment.app.FragmentController.dispatchOptionsItemSelected(FragmentController.java:412) 
        at androidx.fragment.app.FragmentActivity.onMenuItemSelected(FragmentActivity.java:389) 
        at androidx.appcompat.app.AppCompatActivity.onMenuItemSelected(AppCompatActivity.java:228) 
        at androidx.appcompat.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:109) 
        at androidx.appcompat.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:109) 
        at androidx.appcompat.app.ToolbarActionBar$2.onMenuItemClick(ToolbarActionBar.java:64) 
        at androidx.appcompat.widget.Toolbar$1.onMenuItemClick(Toolbar.java:207) 
        at androidx.appcompat.widget.ActionMenuView$MenuBuilderCallback.onMenuItemSelected(ActionMenuView.java:779) 
        at androidx.appcompat.view.menu.MenuBuilder.dispatchMenuItemSelected(MenuBuilder.java:834) 
        at androidx.appcompat.view.menu.MenuItemImpl.invoke(MenuItemImpl.java:158) 
        at androidx.appcompat.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:985) 
        at androidx.appcompat.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:975) 
        at androidx.appcompat.widget.ActionMenuView.invokeItem(ActionMenuView.java:623) 
        at androidx.appcompat.view.menu.ActionMenuItemView.onClick(ActionMenuItemView.java:151) 
        at android.view.View.performClick(View.java:5204) 
        at android.view.View$PerformClick.run(View.java:21153) 
        at android.os.Handler.handleCallback(Handler.java:739) 
        at android.os.Handler.dispatchMessage(Handler.java:95) 
        at android.os.Looper.loop(Looper.java:148) 
        at android.app.ActivityThread.main(ActivityThread.java:5417) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
     Caused by: android.content.res.Resources$NotFoundException: File res/color/filter_dialog_background_light.xml from drawable resource ID #0x7f060089
        at android.content.res.Resources.loadDrawableForCookie(Resources.java:2640)
        at android.content.res.Resources.loadDrawable(Resources.java:2540)
        at android.content.res.TypedArray.getDrawable(TypedArray.java:870)
        at android.view.View.<init>(View.java:3954)
        at android.widget.TextView.<init>(TextView.java:677)
        at android.widget.Button.<init>(Button.java:109)
        at android.widget.CompoundButton.<init>(CompoundButton.java:84)
    	at
09-17 22:29:39.447 23478-23478/de.danoeh.antennapod.debug I/Process: Sending signal. PID: 23478 SIG: 9
```

--- a/core/src/main/res/color/filter_dialog_background_dark.xml
+++ b/core/src/main/res/color/filter_dialog_background_dark.xml
@@ -1,5 +1,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<selector xmlns:android="http://schemas.android.com/apk/res/android">
-    <item android:color="@color/accent_dark" android:state_checked="true"/>
-    <item android:color="@color/dialog_filter_inactive_dark" />
-</selector>

--- a/core/src/main/res/color/filter_dialog_background_light.xml
+++ b/core/src/main/res/color/filter_dialog_background_light.xml
@@ -1,5 +1,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<selector xmlns:android="http://schemas.android.com/apk/res/android">
-    <item android:color="@color/accent_light" android:state_checked="true" />
-    <item android:color="@color/dialog_filter_inactive_light" />
-</selector>

--- a/core/src/main/res/drawable/filter_dialog_background_dark.xml
+++ b/core/src/main/res/drawable/filter_dialog_background_dark.xml
@@ -1,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:drawable="@color/accent_dark" android:state_checked="true"/>
+    <item android:drawable="@color/dialog_filter_inactive_dark" />
+</selector>

--- a/core/src/main/res/drawable/filter_dialog_background_light.xml
+++ b/core/src/main/res/drawable/filter_dialog_background_light.xml
@@ -1,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:drawable="@color/accent_light" android:state_checked="true" />
+    <item android:drawable="@color/dialog_filter_inactive_light" />
+</selector>

--- a/core/src/main/res/values/styles.xml
+++ b/core/src/main/res/values/styles.xml
@@ -74,7 +74,7 @@
         <item name="ic_volume_adaption">@drawable/ic_volume_adaption_black</item>
         <item name="scrollbar_thumb">@drawable/scrollbar_thumb_light</item>
         <item name="filter_dialog_clear">@color/filter_dialog_clear_light</item>
-        <item name="filter_dialog_button_background">@color/filter_dialog_background_light</item>
+        <item name="filter_dialog_button_background">@drawable/filter_dialog_background_light</item>
     </style>
 
     <style name="Theme.AntennaPod.Dark" parent="Theme.Base.AntennaPod.Dark">
@@ -151,7 +151,7 @@
         <item name="ic_volume_adaption">@drawable/ic_volume_adaption_white</item>
         <item name="scrollbar_thumb">@drawable/scrollbar_thumb_dark</item>
         <item name="filter_dialog_clear">@color/filter_dialog_clear_dark</item>
-        <item name="filter_dialog_button_background">@color/filter_dialog_background_dark</item>
+        <item name="filter_dialog_button_background">@drawable/filter_dialog_background_dark</item>
     </style>
 
     <style name="Theme.AntennaPod.TrueBlack" parent="Theme.Base.AntennaPod.TrueBlack">

